src/schema/meta/expression_tests.yaml

```diff
@@ -45,6 +45,10 @@
   result: false
 - expression: '"VolumeTiming" in null'
   result: false
+- expression: exists(null, "bids-uri")
+  result: false
+- expression: exists([], null)
+  result: false
```

Comment:
I assume we want these to be `false` for consistency, but `exists([], *)` seems like it could safely evaluate true, and so `exists(null, *)` could also evaluate true. It's a degenerate `and()`/`product()`.